app/shared/resources/BaseUrls/BaseUrls.js

```diff
@@ -1,6 +1,6 @@
-export const poeFetchAPI = "https://www.pathofexile.com/api/trade/fetch";
+export const poeFetchAPI = "https://www.pathofexile.com/api/trade2/fetch";
 
-export const poeWsUri = "wss://www.pathofexile.com/api/trade/live";
+export const poeWsUri = "wss://www.pathofexile.com/api/trade2/live";
```

Comment:
This would break compatibility with poe 1, would need to select the 1 vs 2 URL dynamically.